src/reducers/projectsReducer.js

```diff
@@ -8,7 +8,9 @@ import {
   DELETE_PROJECT,
   CLOSE_PROJECT,
   REACTIVATE_PROJECT, 
-  CHANGE_PROJECT_SKILL
+  CHANGE_PROJECT_SKILL,
+  ADD_FEEDBACK,
+  GET_FEEDBACKS
 } from "../constants";
```

Comment:
I once told Michał to fix this, but he apparently ignored me. File with action types should be called actionTypes. 
"Constants" is confusing.